x/nodes/types/errors.go

```diff
@@ -95,6 +95,10 @@ func ErrMinimumEditStake(codespace sdk.CodespaceType) sdk.Error {
 	return sdk.NewError(codespace, CodeMinimumEditStake, "validator must edit stake with a stake greater than or equal to current stake")
 }
 
+func ErrSameBinEditStake(codespace sdk.CodespaceType) sdk.Error {
+	return sdk.NewError(codespace, CodeMinimumEditStake, "validator must edit stake to a greater stake Bin tha the current stake")
```

Comment:
There's a typo in the error message: `Bin tha the` should be `Bin than the`

Reply:
fixed in 26ccf1c7c1fb26c47ea42acedf946984b7241b33